plugins/ssh-plugin/src/ssh-plugin-backend.ts

```diff
@@ -271,7 +271,7 @@ const uploadPrivateKey = async (sshkeyManager: SshKeyManager) => {
     }
 
     unlinkSync(privateKeyPath.path);
-    rmdirSync(tempDir, { recursive: true });
+    removeSync(tempDir);
```

Comment:
Are changes for `ssh-plugin` included by mistake in this PR?

Reply:
No, otherwise, the code does not compile

Reply:
Or it didn't until Florent merge his recent PR.